### PR TITLE
Add docstring on hover tooltip

### PR DESCRIPTION
**Pull request by izikl**

### CODE REVIEW

File: src/languageService/kustoLanguageService.ts [hunk 1]
```diff
@@ -1437,12 +1442,14 @@ class KustoLanguageService implements LanguageService {
...
 
             // TODO: handle outputColumns (right now it doesn't seem to be implemented for any function).
-            return new sym.FunctionSymbol.$ctor16(fn.name, fn.body, parameters, null);
+            return new sym.FunctionSymbol.$ctor16(fn.name, fn.body, parameters, fn.docstring);
         };
 
         const createTableSymbol: (tbl: s.Table) => sym.TableSymbol = (tbl) => {
             const columnSymbols = tbl.columns.map((col) => KustoLanguageService.createColumnSymbol(col));
-            return new sym.TableSymbol.$ctor3(tbl.name, columnSymbols);
+            const symbol = new sym.TableSymbol.$ctor3(tbl.name, columnSymbols);
```
Comment (lidermanrony) [inline src/languageService/kustoLanguageService.ts:1450]: can clusters / databases / columns have a docstring as well? we should support anyone that does.

File: src/languageService/kustoLanguageService.ts [hunk 2]
```diff
@@ -1196,7 +1198,8 @@ class KustoLanguageService implements LanguageService {
             return Promise.as(undefined);
         }
 
-        return Promise.as({ contents: quickInfo.Text });
+        // \n\n doesn't work in a markdown, it won't create an empty line. So instead use "\n\n&nbsp;\n\n".
+        return Promise.as({ contents: quickInfo.Text.replace("\n\n", "\n\n&nbsp;\n\n") });
```
Comment (lidermanrony) [inline src/languageService/kustoLanguageService.ts:1202]: can this be null or undefined? if so we can crash here

Comment (izikl) [reply src/languageService/kustoLanguageService.ts:1202]: The previous line is `if (!quickInfo || !quickInfo.Text) {`

File: src/languageService/kustoLanguageService.ts (hunk 1 above)
Comment (izikl) [reply src/languageService/kustoLanguageService.ts:1451]: Added column support but the c# code doesn't support clusters and databases.

**Pull request merged**